posthog-core/src/index.ts

```diff
@@ -1439,6 +1448,10 @@ export abstract class PostHogCore extends PostHogCoreStateless {
               this.reloadFeatureFlags()
             }
 
+            if (!response.supportedCompression?.includes(Compression.GZipJS)) {
```

Comment:
we should check for lz64 here too

Reply:
nvm, https://github.com/PostHog/posthog/blob/bcc624b7150124354edb4cd1f603cf73ce1dbfc9/posthog/api/decide.py#L118-L119
its hardcoded and the API does not return `lz64` here